docs/source/02_installation.md

```diff
@@ -54,3 +54,31 @@ Once this alias is in place, you can run ADAM by simply typing `adam-submit` at
 ```bash
 $ adam-submit
 ```
+
+## Building for Python {#python-build}
+
+To build and test [ADAM's Python bindings](#python), enable the `python`
+profile:
+
+```bash
+mvn -Ppython package
+```
+
+This will enable the `adam-python` module as part of the ADAM build. This module
+uses Maven to invoke a Makefile that builds a Python egg and runs tests. To
+build this module, we require either an active [Conda](https://conda.io/) or
+[virtualenv](https://virtualenv.pypa.io/en/stable/) environment. Additionally,
```

Comment:
I'm not a python guy so I don't know what either of these means.  Is there a one or two-liner we could include here?

```bash
$ mvn -Ppython package
...
[INFO] --- exec-maven-plugin:1.5.0:exec (dev-python) @ adam-python_2.10 ---
pip install -e .
Obtaining file:///Users/heuermh/working/adam/adam-python
Installing collected packages: bdgenomics.adam
  Running setup.py develop for bdgenomics.adam
Successfully installed bdgenomics.adam
[INFO] 

...

[INFO] --- exec-maven-plugin:1.5.0:exec (test-python) @ adam-python_2.10 ---
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named pytest
Build requirements are missing. Run 'make prepare' to install them.
make: *** [check_build_reqs] Error 1
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 2 (Exit value: 2)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
```